packages/big-design/src/components/FileUploader/spec.tsx

```diff
@@ -372,6 +372,29 @@ describe('FileUploader', () => {
 
     expect(screen.getAllByText('File name is too long')).toHaveLength(15);
   });
+
+  it('renders additional action', async () => {
```

Comment:
Would be nice to have some additional tests around not being able to pass styles or different button variants into `dropzoneConfig.action`

Reply:
I have already covered.